cdap-api/src/main/java/co/cask/cdap/api/data/stream/StreamBatchReadable.java

```diff
@@ -47,12 +49,14 @@
  * <pre><code>
  *   class MyMapReduce implements MapReduce {
  *       public void beforeSubmit(MapReduceContext context) {
- *         context.setInput(new StreamBatchReadable("mystream"));
+ *         context.addInput(Input.ofStream(new StreamBatchReadable("mystream")));
```

Comment:
this comment should use the public ofStream(0 factory method. The StreamBatchReadable one is private, and the class is deprecated,